docs/general/run-a-node/prerequisites/hardware-recommendations.md

```diff
@@ -1,218 +1,195 @@
 # Hardware Requirements
 
-The Oasis Network is composed of multiple classes of nodes that participate in
-different committees.
+The Oasis Network is composed of multiple classes of nodes and services, such
+as:
+
+* Consensus Validator/Non-validator Node
+* Emerald ParaTime Compute/Client Node
+* Emerald Web3 Gateway with PostgreSQL
+* Cipher ParaTime Compute/Client Node
 
 This page describes the **minimum** and **recommended** system hardware
 requirements for running different types of nodes on the Oasis Network.
 
 :::caution
 
-If you configure a system with less resources than the recommended values, you
-run the risk of being underprovisioned and causing proposer node timeouts.
-This could result in losing stake.
+If you configure a system with slower resources than the recommended values, you
+run the risk of being underprovisioned, causing proposer node timeouts and
+synchronization delays. This could result in losing stake and not participating
+in committees.
+
+If the system will not have enough memory or storage, that will result in the
+Oasis node process being forcefully killed. This could lead to BadgerDB state
+corruption, losing stake and not participating in committees.
 
 :::
 
-## Consensus Nodes <a id="suggested-minimum-configurations"></a>
 
-To run a non-validator or a validator consensus node, your system should meet
-the following requirements:
+### CPU <a id="suggested-minimum-configurations"></a>
 
-### CPU
+* Consensus Validator/Non-validator Node:
+  * Minimum: 2.0 GHz x86-64 CPU with [AES instruction set] support
+  * Recommended: 2.0+ GH+ x86-64 CPU with 2 cores/vCPUs with
+    [AES instruction set] and [AVX2] support
+ 
+* Emerald ParaTime Compute Node and all ParaTime Client Nodes:
+  * Minimum: 2.0 GHz x86-64 CPU with [AES instruction set] support
+  * Recommended: 2.0+ GHz x86-64 CPU with 4+ cores/vCPUs with
+    [AES instruction set] and [AVX2] support
 
-* Minimum: 2.0 GHz x86-64 CPU with [AES instruction set] support
-* Recommended: 2.0 GHz+ x86-64 CPU with 2 cores or 2 such virtual CPUs with
-  [AES instruction set] and [AVX2] support
+* Emerald Web3 Gateway with PostgreSQL:
+  * Minimum: 2.0 GHz x86-64 CPU
+  * Recommended: 2.0+ GHz x86-64 CPU with 2 cores/vCPUs
 
-:::caution
+* Cipher ParaTime Compute Node:
+  * Minimum: 2.0 GHz x86-64 CPU with [AES instruction set] and [Intel SGX] support
+  * Recommended: 2.0+ GHz x86-64 CPU with 2+ cores/vCPUs with
+    [AES instruction set], [Intel SGX] and [AVX2] support
+
+:::info
+
+During ordinary operation nodes can work with the minimal CPU resources.
```

Comment:
I would use *node* in a personal context.

```suggestion
During regular workload your node will operate with the minimal CPU resources.
```

Reply:
Much better. :+1: